src/events.rs

```diff
@@ -97,6 +96,16 @@ pub struct clap_event_param_mod {
 unsafe impl Send for clap_event_param_mod {}
 unsafe impl Sync for clap_event_param_mod {}
 
+#[repr(C)]
+#[derive(Copy, Clone)]
+pub struct clap_event_param_gesture {
+    pub header: clap_event_header,
+    pub param_id: clap_id,
+}
+
+unsafe impl Send for clap_event_param_gesture {}
+unsafe impl Sync for clap_event_param_gesture {}
```

Comment:
I don't think these are necessary; everything in `clap_event_param_gesture` is already `Send + Sync`.

Reply:
You're right, they are.